BP/common/council_tasks/00_court_chaplain_tasks.txt

```diff
@@ -712,7 +712,43 @@ task_conversion = {
 				desc = COURT_CHAPLAIN_CONVERSION_HOLY_SITE_CORDOBA
 			}
 		}
+		
+			#WtWSMS new ones 
+		
+		if = {
+			limit = {
+				exists = scope:county
+				scope:councillor.faith = {
+					controls_holy_site = vaspurakan
+				}
+				title:c_vaspurakan = {
+					is_holy_site_of = scope:councillor.faith
+				}
+				scope:county.culture = scope:councillor.liege.culture
+			}
+			add = {
+				value = scope:councillor.conversion_holy_site_vaspurakan
+				desc = COURT_CHAPLAIN_CONVERSION_HOLY_SITE_VASPURAKAN
+			}
+		}
 
+		if = {
+			limit = {
+				exists = scope:county
+				scope:councillor.faith = {
+					controls_holy_site = vannes
+				}
+				title:c_vannes = {
+					is_holy_site_of = scope:councillor.faith
+				}
+				scope:county.culture = scope:celtic_group
```

Comment:
How would you change it, precisely? The purpose of celtic paganism is for it to be spreaded mostly amongst celts, isn't it? I can add more cultures to this which were historically celtic pagan, like some Iberian peninsula cultures and celt remnants in the other parts of Europe, where they aren't the part of the group, but I really don't know how to do this.

Reply:
I understand where you are getting but I agree with Lt-Rascek, we can't restrict it to just Celtic culture Celtic pagans, the religion has to be autonomous from culture. If somehow a Persian character becomes Celtic religion that should work out, even if unlikely.

Reply:
Well the reason I wanted it to work like this because I wanted it to work like Iona holy site but more specific and more wide at the same time, but I guess I can make it just if the liege culture is matching with the county culture and reduce uniqueness of this holy site to none.

Reply:
Ok, I see. If you want to replace this effect by another which is more unique no problems, but I think tying the unique content with specific culture groups is problematic in terms of gameplay.

Reply:
Well celtic paganism was completely tied with celtic culture and it was specific religion of those ethnicities with its completely unique abilities, nearly no one changed their religion to this, and vistula venedi, for example, are descendants of one of the central european celtic cultures, that's why they could remain celtic in terms of religion but not in ones of language. I guess I will just tie other thing for this, though I don't know which one.

Reply:
I think I got it - 20% bonus for Celtic group provinces, 20% bonus for if liege's and county's culture are the same, and Celtic rulers will receive 40% total on their culture provinces while all the others - 20%

Reply:
Sounds good to me